### DISCUSSION TITLE
Why does the extension need my browsing history?

### DISCUSSION THREAD

**Discussion by useroute** [tarampampam/random-user-agent#561] [Q&A]
Hi, I have noticed "Random User Agent Switcher" has my browsing history, why is that?

**Comment by tarampampam** [comment]
All required permissions are defined [here](https://github.com/tarampampam/random-user-agent/blob/26f5ef9c8f5a4c3465f013644000cc78d8737e41/public/manifest.json#L37-L43).

As you can see, "browser history" is not listed here, and it's not required for this extension to work. However, since the extension works with all opened URLs, perhaps that's why the notice is shown?
